src/Console/ServeCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace TightenCo\Jigsaw\Console;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ServeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('serve')
+            ->setDescription('Serve local site with php built-in server.')
+            ->addArgument(
```

Comment:
That makes sense to me. I'll make the change to this shortly. Do you want me to submit another PR for updating the build command or are you going to do that yourself?